### DISCUSSION TITLE
Bind Steam Games to Monitor 1

### DISCUSSION THREAD

**Discussion by truensado** [hyprwm/Hyprland#6388] [General]
As the title states I'm trying to find steam games to my monitor 1 when they launch. I've had success doing this using these rules:

"windowrulev2 = fullscreen,class:^steam_app\d+$
windowrulev2 = monitor 1,class:^steam_app_\d+$
workspace = 1, border:false, rounding:false" 

Even though this does work, my issue is that I want to be able to use my second monitor while the game launches on my monitor 1, however if I leave my mouse on the second monitor, the game will originally launch on my main monitor 1, but then after launching for a couple of seconds it will teleport to my second monitor because mouse is there. Also when it teleports to my second monitor, the area it was in on my first monitor get a blurry outline of where it was and when I run my cursor through my first monitor it begins to duplicate many times over. Thank you in advance for everybody's help. Below is an image of the glitch.

![240609_13h38m46s_screenshot](https://github.com/hyprwm/Hyprland/assets/171737294/45f934fb-f961-4564-a74e-ad662ec58bd0)

**Comment by kumarukemuri** [comment]
Try windowrulev2 = workspace 1, initialclass:^(steam_app_.*)$, works for me

Also need to add workspace = 1, monitor:(whatever monitor it is)
